client/src/pages/Profile.js

```diff
@@ -88,146 +89,499 @@ const useStyles = makeStyles(theme => ({
+            ],
+            "title": "test post1",
+            "description": "tesing posts",
+            "link": "test",
+            "user": "5d4c3fcb1de1e451e8c1760d",
+            "image": "https://team-pineapple.s3.ca-central-1.amazonaws.com/1565283929383.jpg",
+            "date": "2019-08-08T17:05:29.657Z",
+            "__v": 0
+        },
+        {
+            "_id": "5d4c55cacf7d2046c003baf0",
+            "tags": [
+                "['']"
+            ],
+            "title": "test post",
+            "description": "tesing posts",
+            "link": "test",
+            "user": "5d4c3fcb1de1e451e8c1760d",
+            "image": "https://team-pineapple.s3.ca-central-1.amazonaws.com/1565283786119.jpg",
+            "date": "2019-08-08T17:03:06.912Z",
+            "__v": 0
+        },
+        {
+            "_id": "5d4c55cacf7d2046c003baf0",
+            "tags": [
+                "['']"
+            ],
+            "title": "test post",
+            "description": "tesing posts",
+            "link": "test",
+            "user": "5d4c3fcb1de1e451e8c1760d",
+            "image": "https://team-pineapple.s3.ca-central-1.amazonaws.com/1565283786119.jpg",
+            "date": "2019-08-08T17:03:06.912Z",
+            "__v": 0
+        },
+        {
+            "_id": "5d4c55cacf7d2046c003baf0",
+            "tags": [
+                "['']"
+            ],
+            "title": "test post",
+            "description": "tesing posts",
+            "link": "test",
+            "user": "5d4c3fcb1de1e451e8c1760d",
+            "image": "https://team-pineapple.s3.ca-central-1.amazonaws.com/1565283786119.jpg",
+            "date": "2019-08-08T17:03:06.912Z",
+            "__v": 0
+        },
+        {
+            "_id": "5d4c55cacf7d2046c003baf0",
+            "tags": [
+                "['']"
+            ],
+            "title": "test post",
+            "description": "tesing posts",
+            "link": "test",
+            "user": "5d4c3fcb1de1e451e8c1760d",
+            "image": "https://team-pineapple.s3.ca-central-1.amazonaws.com/1565283786119.jpg",
+            "date": "2019-08-08T17:03:06.912Z",
+            "__v": 0
+        },
+        {
+            "_id": "5d4c55cacf7d2046c003baf0",
+            "tags": [
+                "['']"
+            ],
+            "title": "test post",
+            "description": "tesing posts",
+            "link": "test",
+            "user": "5d4c3fcb1de1e451e8c1760d",
+            "image": "https://team-pineapple.s3.ca-central-1.amazonaws.com/1565283786119.jpg",
+            "date": "2019-08-08T17:03:06.912Z",
+            "__v": 0
+        },
+    ])
+    
+    // useEffect(() => {
+    //     // fetch(`users/${username}`)
+    //     // .then(response => response.json())
+    //     // .then(data => setBoards((boards = data['user']['boards'])))
+    //     // .then(data => setPosts((posts = data['user']['posts'])))
+    // })
+    
+    if (boards.length === 0 && posts.length === 0) {
```

Comment:
there is a lot of repeated code based on how you did this.... I feel like the checking on boards and posts should happen in the return and should be more specific

Reply:
yeah I'm trying to figure out another way to do this. I'm working on a fix now